src/main/java/com/es/phoneshop/model/product/service/ServiceException.java

```diff
@@ -0,0 +1,19 @@
+package com.es.phoneshop.model.product.service;
+
+public class ServiceException extends Exception {
```

Comment:
ServiceException is too abstract name, and you should give appropriate names to sustain self-documentability of your code. Rename this one or add some subclasses with exact names.